surfer/viz.py

```diff
@@ -30,6 +32,12 @@
 logger = logging.getLogger('surfer')
 
 
+# Monkey-patch for Mayavi 4.5: filters are missing a point_data attribute that
+# Threshold accesses on initialization
+if LooseVersion(mayavi.__version__) <= LooseVersion('4.5.0'):
+    mayavi.filters.api.Threshold._get_data_range = lambda x: []
```

Comment:
Yeah it's heavy-handed but probably our only option.

One other thing we could do that would be less brutal is make a context manager that *temporarily* monkey-patches, and then sets back once the potentially offensive line completes, like:
```
with PatchedMayavi():
    surf = ...
```